pombola/core/tests/test_organisation_histories.py

```diff
@@ -0,0 +1,52 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from pombola.core import models
+
+
+class OrganisationHistoryTest(TestCase):
```

Comment:
Looking good - could you add this one as well?  you're missing a test for the same on committee messages page https://www.pa.org.za/organisation/portfolio-committee-on-agriculture-forestry-and-fisheries/messages/